packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -398,6 +400,24 @@ class _HelperErrorState extends State<_HelperError> with SingleTickerProviderSta
     );
   }
 
+  Widget _buildErrorWidget() {
```

Comment:
Nit: I wonder if we can just put this logic in `_buildError()`.

Something like :

```dart
child : widget.errorWidget ?? Text(
            widget.errorText!,
            style: widget.errorStyle,
            textAlign: widget.textAlign,
            overflow: TextOverflow.ellipsis,
            maxLines: widget.errorMaxLines,
          ),
```

Reply:
Ahhmmm Yes you are right we can do this. 
Thanks! :)